packages/framer-motion/src/utils/use-instant-transition.ts

```diff
@@ -37,3 +37,7 @@ export function useInstantTransition() {
         })
     }
 }
+
+export function disableInstantTransitions() {
```

Comment:
I thought it made more sense to call this `disableInstantTransitions` that `enableAnimations` because I thought that would be a bit too broad as a public API and might be misinterpreted

Reply:
Totally agree